coramin/utils/pyomo_utils.py

```diff
@@ -0,0 +1,21 @@
+import pyomo.environ as pe
+
+
+def get_obj(m):
```

Comment:
This is a style thing, but I read this as "get object".  Would you be opposed to renaming it `get_objective()`?